ClearKeep/Scenes/Home/HomeView.swift

```diff
@@ -28,6 +28,7 @@ struct HomeView: View {
 	@Environment(\.injected) private var injected: DIContainer
 	@Environment(\.joinServerClosure) private var joinServerClosure: JoinServerClosure
 	@State private var selectedGroupId: Int64 = 0
+    @State private var selectedNotiDomain: String = ""
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/0b09249675dff5b0a460b69f27e9f3e1d518728a/ClearKeep/Scenes/Home/HomeView.swift#L31"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>HomeView.swift:31</code></td>
    </tr>
  </tbody>
</table>